### ISSUE TITLE
False positive: No void expression

### ISSUE THREAD

**Issue by VincentLanglet** [palantir/tslint#4556]
### Bug Report

- __TSLint version__: 5.13.0
- __TypeScript version__: 3.3.3333
- __Running TSLint via__: (pick one) CLI

#### TypeScript code being linted

```ts
!!data ? console.info(message, data) : console.info(message);
```

with `tslint.json` configuration:

```json
"no-void-expression": [true, "ignore-arrow-function-shorthand"],
```

#### Actual behavior
Error

#### Expected behavior
No error
